src/ros_can_nodes/scripts/generate_message_properties_map.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
```

Comment:
when I build on my computer I don't get any messages in the output only

```

/* Auto-generated for ros_can_nodes */
/* DO NOT EDIT */
#ifndef MESSAGE_PROPERTIES_MAP_HPP
#define MESSAGE_PROPERTIES_MAP_HPP

#include <unordered_map>
#include <string>

struct message_properties {
    const std::string definition;
    const std::string md5sum;
};

extern const std::unordered_map<std::string, const message_properties> message_properties_map;

#endif // MESSAGE_PROPERTIES_MAP_HPP
```

Reply:
it should have also generated message_properties_map.cpp in src/

Reply:
new changes generates files into build dir